superset_config.py

```diff
@@ -29,4 +29,15 @@
 AUTH_USER_REGISTRATION = True
 
 # The default user self registration role
-AUTH_USER_REGISTRATION_ROLE = "Gamma"
\ No newline at end of file
+AUTH_USER_REGISTRATION_ROLE = "Gamma"
+
+# Any other additional roles to be assigned to the user on top of the base role
+# Note: by design we cannot use AUTH_USER_REGISTRATION_ROLE to
+# specify more than one role
+AUTH_USER_ADDITIONAL_ROLES = ["sql_lab"]
+
+try:
+    # Overwrite repo settings with local settings
+    from local_settings import *
```

Comment:
Good idea, could name it as local_config to keep it consistent with flask/superset terminology.

Reply:
Agree!